src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php

```diff
@@ -72,9 +64,8 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, callable $va
      *
      * @param array|callable $labelTranslationParameters The parameters used to translate the choice labels
```

Comment:
let's remove as many annotations as possible, and add the union types meanwhile!

Reply:
Agreed, but I prefer to do that in a different PR.